### ISSUE TITLE
Ad blocker for Get points for free.

### ISSUE THREAD

**Issue by AkshayJambagi** [aviraxp/AdBlocker_Reborn#36]
I have been lately come across the problem of ads where in some apps if u want to get extra points or in game extra life u have to watch a video to get the benifit.

I was wondering if you could bypass the video duration values and skip the long time video seeing ads

**Comment by eebssk1** [comment]
Yep..Especially some app force user to watch ads to unlock something..
